src/Adapters/Cache/RedisCacheAdapter.js

```diff
@@ -1,6 +1,8 @@
 import redis from 'redis';
 import logger from '../../logger';
 
+const DEFAULT_REDIS_TTL = 30 * 1000; // 30 seconds
```

Comment:
Arg, I had something there initially where if TTL wasn't provided we'd use one but the lockout tests wouldn't pass. I meant to remove that for the time being.